From 97ae739d13e02e5ca7f9b47c40ffddf96eabc566 Mon Sep 17 00:00:00 2001
From: Ankita Garg <ankita@in.ibm.com>
Date: Fri, 19 Sep 2008 08:26:47 -0400
Subject: [PATCH] From ankita@in.ibm.com Wed Sep 10 02:10:05 2008
 Date: Wed, 10 Sep 2008 11:36:24 +0530
 To: Steven Rostedt <rostedt@goodmis.org>
 Cc: LKML <linux-kernel@vger.kernel.org>, RT <linux-rt-users@vger.kernel.org>, Ingo Molnar <mingo@elte.hu>, Thomas Gleixner <tglx@linutronix.de>
 Subject: Re: 2.6.24.7-rt18
 Hi Steven,

On Tue, Sep 09, 2008 at 11:44:42PM -0400, Steven Rostedt wrote:
> We are pleased to announce the 2.6.24.7-rt18 tree, which can be
> downloaded from the location:
>
>   http://rt.et.redhat.com/download/
>
> Information on the RT patch can be found at:
>
>   http://rt.wiki.kernel.org/index.php/Main_Page
>
> Changes since 2.6.24.7-rt17
>
>   - seqlock serialize writes (Gregory Haskins)
>
>   - Fix above patch that broke non PREEMPT_RT (Steven Rostedt)
>
>   - Add sysctl to warn on RT task using non PI futex (Steven Rostedt)
>

A trivial patch to remove the hard coded value of MAX_RT_PRIO in the
above patch.

Signed-off-by: Ankita Garg <ankita@in.ibm.com>
---
 kernel/futex.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/kernel/futex.c b/kernel/futex.c
index 221c7aa..b88d7ed 100644
--- a/kernel/futex.c
+++ b/kernel/futex.c
@@ -1241,7 +1241,7 @@ static int futex_wait(u32 __user *uaddr, struct rw_semaphore *fshared,
 			       "RT task %s:%d with priority %d"
 			       " using non PI futex\n",
 			       current->comm, current->pid,
-			       100 - current->prio);
+			       MAX_RT_PRIO - current->prio);
 		}
 	}
 
-- 
1.6.0.90.g436ed

